prac_05/wimbledon.py

```diff
@@ -0,0 +1,57 @@
+"""
+Wimbledon stat tracker
+Estimate: 70 minutes
+Actual:   75 minutes
+"""
+
+FIELD_TITLES = 1
+WINNING_COUNTRY_COLUMN = 1
+WINNER_COLUMN = 2
+FILENAME = "wimbledon.csv"
+
+
+def main():
+    """Program to find the amount of times a player has won Wimbledon and the winning countries."""
+    with open(FILENAME, "r", encoding="utf-8-sig") as in_file:
+        all_years_data = [line.strip() for line in in_file.readlines()]
+        year_data = [year.split(",") for year in all_years_data[FIELD_TITLES:]]
+        print("Wimbledon Champions:")
+        winners = find_winners(year_data)
+        players_to_wins = calculate_player_wins(winners)
+        for players, wins in players_to_wins.items():
+            print(f"{players}: {wins}")
```

Comment:
The example output doesn't use colons, which isn't really relevant to the actual quality of the code, but wanted to point it out in case you want to exactly match the example formatting.